Clarifying use of centralized configuration options (#6622)
wazuh/wazuh-documentation: source/user-manual/reference/centralized-configuration.rst

+-------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+

> | **name**    | Allows assignment of the block to one particular agent.                                                                                                           |

Reviewer: ```suggestion
| **name**    | Assigns the block to agents with specific names.                                                                                                                  |
```

+             +-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
|             | Allowed values                                        | Any regular expression that matches the agent name .                                                      |
+-------------+-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+

> | **os**      | Allows assignment of the block to an operating system.                                                                                                            |

Reviewer: ```suggestion
| **os**      | Assigns the block to agents on specific operating systems.                                                                                                        |
```

+             +-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
|             | Allowed values                                        | Any regular expression that matches the agent OS information.                                             |
+-------------+-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+

> | **profile** | Allows assignment of a profile name to a block. Any agent configured to use the defined :ref:`profile <reference_ossec_client_config_profile>` may use the block. |

Reviewer: ```suggestion
| **profile** | Assigns the block to agents with specific profiles as defined in:ref:`reference_ossec_client_config_profile`.                                                     |
```

Author: I changed a little bit this proposal because it was not showing properly the reference after compilation.

+             +-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+

> |             | Allowed values                                        | Any regular expression that matches the defined profile.                                                  |

Reviewer: ```suggestion
|             | Allowed values                                        | Any regular expression that matches the agent profile.                                                    |
```

+-------------+-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+

> .. topic:: Example

Reviewer: We could have a more varied display of regex possibilities. For example `^agent(11|12|13)`

Author: The regular expression used for this is very limited, I used the one that worked. 
![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/74bf3a1e-ad55-41ba-92f9-ba34debddf23)

![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/b108fece-b9e7-483a-98a2-7c24e6da16b2)

![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/ddf4c883-fdfe-48ce-ba4a-abde2839bfc6)

	.. code-block:: xml

		<agent_config name=”^agent01”>
		...
		<agent_config os="^Linux">
		...
		<agent_config profile="^UnixHost">

> Get the agent name and operating system.

Reviewer: ```suggestion
   To get the agent name and operating system information, you can run the ``agent_control`` utility.
```
